cmd/rhobs/requests.go

```
@@ -0,0 +1,298 @@
+}
+
+func printMetricsAsCsv(metrics []rhobsMetric) {
+	columns := getTableColumns(metrics)
+
+	writer := csv.NewWriter(os.Stdout)
+
+	header := []string{}
+	for _, column := range columns {
+		header = append(header, column.name)
+	}
+	writer.Write(header)
```

golangci-lint (2.11.4)

[error] Error return value of `writer.Write` is not checked

(errcheck)